common/reviews/api/api-extractor.api.ts

```diff
@@ -0,0 +1,76 @@
+class Analyzer {
```

Comment:
A bunch of these items probably don't belong in the public API (or would need to be cleaned up significantly).  Also we are missing a bunch of JSDoc documentation.

We can handle that in a separate PR.